reoptjl/views.py

```diff
@@ -810,6 +821,8 @@ def summary(request, user_uuid):
             run_uuid__in=[i.run_uuid for i in UserUnlinkedRuns.objects.filter(user_uuid=user_uuid)]
         ).only(
             'run_uuid',
+            'user_uuid',
```

Comment:
Added these two fields in Summary response for informational and validation purposes. The UI can ignore these.